librz/magic/file.h

```diff
@@ -46,9 +46,6 @@
 /* Do this here and now, because struct stat gets re-defined on solaris */
 #include <sys/stat.h>
 #include <stdarg.h>
-#ifdef _MSC_VER
```

Comment:
One more embedded file for https://github.com/rizinorg/rizin/issues/209

Reply:
Yeah, however I am not sure how to use this if not embedding. AFAIR this is slightly different from the linux `magic` and I don't think distros have a `magic-bsd` package.

Reply:
Yes, embedding is fine. We just should note imported version somewhere and our changes.